<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registr</class>
 <widget class="QDialog" name="Registr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>820</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift SemiBold SemiConden</family>
    <pointsize>14</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/user.png</normaloff>:/img/user.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background: qlineargradient(x1:1, y1:0, x2:0, y2:0, 
                                  stop:0 rgba(2,0,36,1), 
                                  stop:0.36 rgba(171,82,9,1), 
                                  stop:0.97 rgba(8,62,148,1));
color:rgb(211, 211, 211);
}

QLineEdit{
	background-color:rgb(232, 232, 232);
	border: 1px solid rgb(209, 136, 0);
	border-radius: 18px;
	padding-left: 5px;
	color:rgb(40, 40, 40);
}

QLineEdit:focus{
	border: 2px solid rgb(9, 100, 186);
	background-color: rgb(208, 208, 208)
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <spacer name="horizontalSpacer_10">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QFrame" name="verticalFrame">
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color:#fff
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Franklin Gothic Heavy</family>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#c38200;&quot;&gt;BRY&lt;/span&gt;&lt;span style=&quot; font-size:14pt; color:#0964ba;&quot;&gt;Insurance&lt;/span&gt;&lt;/p&gt;&lt;hr/&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Franklin Gothic Heavy</family>
          <pointsize>26</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
color: rgb(9, 100, 186);

}
</string>
        </property>
        <property name="text">
         <string>SIGN UP</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="surnameLn">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif Collection</family>
            <pointsize>10</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	background-color:rgb(232, 232, 232);
	border: 1px solid rgb(209, 136, 0);
	border-radius: 18px;
	padding-left: 5px;
	color:rgb(40, 40, 40);
	margin-left:17px;
	margin-right:17px;
	margin-top: 10px;
margin-bottom:10px;
}

QLineEdit:focus{
	border: 2px solid rgb(9, 100, 186);
	background-color: rgb(208, 208, 208)
}</string>
          </property>
          <property name="placeholderText">
           <string>Surname:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameLn">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif Collection</family>
            <pointsize>10</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	background-color:rgb(232, 232, 232);
	border: 1px solid rgb(209, 136, 0);
	border-radius: 18px;
	padding-left: 5px;
	color:rgb(40, 40, 40);
margin-left:17px;
	margin-right:17px;
margin-bottom:10px;
}

QLineEdit:focus{
	border: 2px solid rgb(9, 100, 186);
	background-color: rgb(208, 208, 208)
}</string>
          </property>
          <property name="placeholderText">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="phonenumLn">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif Collection</family>
            <pointsize>10</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	background-color:rgb(232, 232, 232);
	border: 1px solid rgb(209, 136, 0);
	border-radius: 18px;
	padding-left: 5px;
	color:rgb(40, 40, 40);
margin-left:17px;
	margin-right:17px;
margin-bottom:10px;
}

QLineEdit:focus{
	border: 2px solid rgb(9, 100, 186);
	background-color: rgb(208, 208, 208)
}</string>
          </property>
          <property name="placeholderText">
           <string>Phone number:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="questionLn">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif Collection</family>
            <pointsize>10</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	background-color:rgb(232, 232, 232);
	border: 1px solid rgb(209, 136, 0);
	border-radius: 18px;
	padding-left: 5px;
	color:rgb(40, 40, 40);
margin-left:17px;
	margin-right:17px;
margin-bottom:10px;
}

QLineEdit:focus{
	border: 2px solid rgb(9, 100, 186);
	background-color: rgb(208, 208, 208)
}</string>
          </property>
          <property name="placeholderText">
           <string>Secret question:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="answearLn">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif Collection</family>
            <pointsize>10</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	background-color:rgb(232, 232, 232);
	border: 1px solid rgb(209, 136, 0);
	border-radius: 18px;
	padding-left: 5px;
	color:rgb(40, 40, 40);
margin-left:17px;
	margin-right:17px;
margin-bottom:10px;
}

QLineEdit:focus{
	border: 2px solid rgb(9, 100, 186);
	background-color: rgb(208, 208, 208)
}</string>
          </property>
          <property name="placeholderText">
           <string>Answer to question:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="usernameLn">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif Collection</family>
            <pointsize>10</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	background-color:rgb(232, 232, 232);
	border: 1px solid rgb(209, 136, 0);
	border-radius: 18px;
	padding-left: 5px;
	color:rgb(40, 40, 40);
margin-left:17px;
	margin-right:17px;
margin-bottom:10px;
}

QLineEdit:focus{
	border: 2px solid rgb(9, 100, 186);
	background-color: rgb(208, 208, 208)
}</string>
          </property>
          <property name="placeholderText">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="passwordLn">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Sans Serif Collection</family>
              <pointsize>10</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	background-color:rgb(232, 232, 232);
	border: 1px solid rgb(209, 136, 0);
	border-radius: 18px;
	padding-left: 5px;
	color:rgb(40, 40, 40);
	margin-left:17px;

}

QLineEdit:focus{
	border: 2px solid rgb(9, 100, 186);
	background-color: rgb(208, 208, 208)
}</string>
            </property>
            <property name="placeholderText">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showPasswordCb">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_17">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="confirmPb">
          <property name="font">
           <font>
            <family>Franklin Gothic Heavy</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border: solid 0px #ffffff;
background-color: #c38200;
padding-top: 6px;
padding-bottom: 6px;
padding-left: 15px;
padding-right: 15px;
border-radius:18px;
margin-left:10px;
margin-right:10px;
color:#fff;
}

QPushButton:hover{
border: solid 2px rgb(0, 0, 0);
background-color: rgb(7, 78, 145);
padding-top: 6px;
padding-bottom: 6px;
padding-left: 15px;
padding-right: 15px;
border-radius:18px;
}</string>
          </property>
          <property name="text">
           <string>Confirm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
